### ISSUE TITLE
Regression in check_unison_image call for 0.5.9

### ISSUE THREAD

**Issue by markglenn** [EugenMayer/docker-sync#611]
### Error/Feature Requestion/Docs

It appears that when the `check_unison_image` method was removed in 0.5.9, the call to it was left in within the `upgrade_check.rb` file.

.../docker-sync-0.5.9/lib/docker-sync/upgrade_check.rb:129:in `check_and_warn': undefined method `check_unison_image' for #<UpdateChecker:0x00007fad7ea4c0f8> (NoMethodError)

See: 
https://github.com/EugenMayer/docker-sync/commit/9ffd23b5a30c63cfe3f057d14a4e433ced6d420c

### Docker Driver
Docker Community Edition - v18.06.1-ce-mac73 (26764)

### Sync strategy

I'm using the default strategy from the docker-sync-boilerplate project, but it appears in the rsync strategy also.

### your docker-sync.yml

    version: "2"

    options:
      verbose: true
    syncs:
      appcode-native-osx-sync: # tip: add -sync and you keep consistent names as a convention
        src: './app'
        # sync_strategy: 'native_osx' # not needed, this is the default now
        sync_excludes: ['ignored_folder', '.ignored_dot_folder']

### OS

macOS High Sierra v10.13.6

**Comment by gpkfr** [comment]
Perhaps as a workaround for end users : 

docker pull docker pull eugenmayer/unison:2.51.2.1

export DOCKER_SYNC_SKIP_UPGRADE=True
docker-sync start

**Comment by EugenMayer** [comment]
fixed with 0.5.10, thank you for the patience

**Comment by EugenMayer** [comment]
@phillipsnick i was aiming to fix https://github.com/EugenMayer/docker-sync/issues/609 first but i had no progress yet

**Comment by carlwolff** [comment]
If anyone still have the issue and forgot to read the [environment variable spec](https://github.com/EugenMayer/docker-sync/issues/611#issuecomment-439648796) above you can also go with:

```bash
$ mv ~/.docker-sync-global.yml ~/.docker-sync-global.yml.bak
$ docker-sync start
```

Note: All our data was harmed in the process 😀(we recreate quite volumes often)

**Comment by andy-viv** [comment]
@EugenMayer thanks for all the hard work!

Any chance 0.5.10 can go out with this fix before #609?

**Comment by andy-viv** [comment]
@EugenMayer looks like this is still an issue in 0.5.10:

```
/Library/Ruby/Gems/2.3.0/gems/docker-sync-0.5.10/lib/docker-sync/upgrade_check.rb:59:in `check_and_warn': undefined method `check_unison_image' for #<UpdateChecker:0x00007fcf5897d860> (NoMethodError)
Did you mean?  check_rsync_image
	from /Library/Ruby/Gems/2.3.0/gems/docker-sync-0.5.10/lib/docker-sync/upgrade_check.rb:18:in `run'
	from /Library/Ruby/Gems/2.3.0/gems/docker-sync-0.5.10/tasks/sync/sync.thor:33:in `start'
	from /Library/Ruby/Gems/2.3.0/gems/thor-0.20.3/lib/thor/command.rb:27:in `run'
	from /Library/Ruby/Gems/2.3.0/gems/thor-0.20.3/lib/thor/invocation.rb:126:in `invoke_command'
	from /Library/Ruby/Gems/2.3.0/gems/thor-0.20.3/lib/thor.rb:387:in `dispatch'
	from /Library/Ruby/Gems/2.3.0/gems/thor-0.20.3/lib/thor/base.rb:466:in `start'
	from /Library/Ruby/Gems/2.3.0/gems/docker-sync-0.5.10/bin/docker-sync:14:in `<top (required)>'
	from /usr/local/bin/docker-sync:22:in `load'
	from /usr/local/bin/docker-sync:22:in `<main>'
```
